includes/documents/class-wcpdf-invoice.php

```diff
@@ -439,6 +439,22 @@ public function init_settings() {
 
 	}
 
+	/**
+	 * Document number title
+	 */
+	public function get_number_title() {
+		// override/not using $this->title to allow for language switching!
```

Comment:
This comment (and the other copies in this PR) seems to have been copied over from the `get_title()` method but AFAIK doesn't apply here?